src/main/java/com/oath/halodb/CompactionManager.java

```diff
@@ -279,13 +280,16 @@ private void copyFreshRecordsToNewFile(int idOfFileToCompact) throws IOException
                     else {
                         // write thread wrote a new version while this version was being compacted.
                         // therefore, this version is stale.
+                        // TODO: From the design, currentWriteFile should be be compacted?
```

Comment:
Why add this comment? It is no relevant code change, better not introduce this kind of comment.

Reply:
Yes, it's another minor issue I found when I adding this feature. I'll address it in another PR.